OpenKh.Tools.BbsEventTableEditor/ViewModels/EventViewModel.cs

```diff
@@ -61,15 +86,26 @@ public byte Room
                 Event.Room = (byte)Math.Min(99, (int)value);
                 OnPropertyChanged(nameof(Name));
                 OnPropertyChanged(nameof(MapPath));
+                OnPropertyChanged(nameof(Rooms));
```

Comment:
**BUG**: You might want to add `OnPropertyChanged(nameof(Rooms));` when the property `World` changes too or the list of rooms will not change.

Reply:
Thanks. I have added this change.